packages/trilinoscouplings/examples/scaling/example_Poisson2D_p2_tpetra.cpp

```diff
@@ -159,8 +159,8 @@ using Teuchos::ParameterList;
 /*********************************************************/
 typedef double scalar_type;
 typedef Teuchos::ScalarTraits<scalar_type> ScalarTraits;
-typedef int local_ordinal_type;
-typedef int global_ordinal_type;
+using local_ordinal_type = Tpetra::Map<>::local_ordinal_type;
+using global_ordinal_type = Tpetra::Map<>::global_ordinal_type;
 typedef KokkosClassic::DefaultNode::DefaultNodeType NO;
```

Comment:
See above.  Why not just get this Node typedef from `Tpetra::Map`?

Reply:
Thanks @mhoemmen !  The response to all your questions is the same - getting the code to build with deprecated code off was a game of wack-a-mole and I only hit the moles that taunted me 😂. After the tester is done I’ll change the node types you mentioned

Reply:
@tjfulle cool no worries :D